src/utils/gateway_oracle.ts

```diff
@@ -0,0 +1,10 @@
+import type { ArweaveOracle } from './arweave_oracle';
+import fetch from 'node-fetch';
```

Comment:
We can probably just eliminate `node-fetch` in this use case, React already has access to a `fetch` method. Then we won't need the types for node-fetch anymore either

Reply:
@fedellen feel free to push commits for this :D